storybook/src/components/Spotlight/Spotlight.stories.tsx

```diff
@@ -29,6 +36,38 @@ type Story = StoryObj<typeof meta>
 
 export const Default: Story = {}
 
+export const StickOut: Story = {
+  render: ({ as, color }) => (
+    <Grid>
+      <Grid.Cell
+        span="all"
+        rowSpan={{ narrow: 2, medium: 2, wide: 1 }}
+        rowStart={2}
+        style={{
+          marginBlock: 'calc(var(--ams-space-grid-md) * -1)',
+          marginInline: 'calc(var(--ams-space-grid-lg) * -1)',
+        }}
```

Comment:
This could become something like an `overlayWhitespace` Boolean prop, although we need a bit more information to correctly select medium and large space lengths.